pom.xml

```diff
@@ -173,7 +175,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <server.jvm.args>${jvm.args.other} ${jvm.args.byteman} ${jvm.args.memory} ${jvm.args.debug} ${jvm.args.modular}</server.jvm.args>
```

Comment:
What's the reason for removing `jvm.args.byteman`?

Reply:
Thanks for your question Manuel, I think that the byteman argument is specific to only some modules. So I would opt to include it only in modules that need it, and not in the root pom. WDYT? Please note that the jvm.args.byteman variable is defined in the root pom too, so it would be injected in all projects using server.jvm.args even when not using byteman.

Reply:
I see your point, thanks. I will update the PR.

Reply:
Which modules use byteman? If there aren't many then my vote would be to have each module set their own system properties since they add noise to command lines.

Reply:
There are quite a few modules using byteman: arjuna, ArjunaJTA, ArjunaJTS, compensation, rts, txbridge and XTS (byteman_support). Though not all sub modules needs it . However I investigated a bit more and apparently some modules (e.g. ArjunaJTA/jta) fail when 'jvm.args.byteman' is set in the the 'server.jvm.args' argument which error `org.jboss.arquillian.container.spi.client.container.LifecycleException: The java process starting the managed server exited unexpectedly with code [1]`, which is , I think, due to the fact that byteman start an agent with a listener (which needs a port, see line https://github.com/jbosstm/narayana/blob/main/pom.xml#L162). I couldn't go deeper in the error though. Anyway I'd rather not use such a configuration generally because it starts an agent which might not be needed and it consumes more resources and time. After this additional consideration, do you agree with me @jmfinelli that it would be better not include it by default as an server argument ?

Reply:
Thanks

Reply:
Thanks @marcosgopen. It looks like we need to investigate further how we activate byteman in different modules. WDYT about opening a new JBTM to track this task? Nevertheless, I agree with you guys that activating byteman from the root pom is a waste of resources thus I'll approve this PR as is.

Reply:
I agree with you Manuel, a new JBTM for tracking the usage of Byteman would be helpful. Thanks